Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Resources.mustache

```diff
@@ -813,14 +832,28 @@ namespace {{ResourceClassesNamespace}}
         {{#Referenced}}
         {{#Property}}
 
+        {{#PropertyIsUnifiedAndLocallyDefined}}
+        private {{PropertyType}} _{{PropertyFieldName}};
+
+        {{/PropertyIsUnifiedAndLocallyDefined}}
         /// <summary>
         /// {{{Description}}}
         /// </summary>
         {{Misc}}
+        {{#PropertyIsUnifiedAndLocallyDefined}}
```

Comment:
These changes are for non-primary key properties.
As of now, there is no resource in the model that would benefit from these changes, but I'm adding them now if one is introduced in the future.